.github/workflows/_cloudflare-r2-release.yml

```
@@ -0,0 +1,562 @@
+          cat << 'EOF'
+          ${{ steps.upload.outputs.file-urls }}
+          EOF
+
+  create-release:
+    name: Create GitHub Release
+    needs: [build, upload]
+    runs-on: ubuntu-latest
+    if: inputs.create_release && startsWith(github.ref, 'refs/tags/') && needs.build.outputs.success == 'true'
+    
+    outputs:
+      release_url: ${{ steps.release.outputs.html_url }}
```

actionlint (1.7.7)

property "html_url" is not defined in object type {assets: string; id: string; upload_url: string; url: string}

(expression)